models/item.js

```diff
@@ -19,7 +19,6 @@ module.exports = function(sequelize, DataTypes) {
           if( params[ key ] !== undefined ) {
             memo[ key ] = params[ key ]
           }
-
```

Comment:
Please add this new line back; let's not make changes in files that we need not make changes in (it confuses the PR).